intercom-cpp/src/posix/dlwrapper.h

```diff
@@ -13,7 +13,7 @@ namespace posix
 {
 
 //!
-class dlopen_wrapper
+class DlWrapper
 {
 public:
```

Comment:
Now that we use capital naming in the class, are we okay with the file name being lower case?

Traditionally I've used upper case names for files containing upper case items. In Rust files contain mods that contain types so all names are lower case for this reason.

In practice I'm indifferent about this - is there preference in Linux world? I know capital characters in file names might be frowned upon in some contexts. Just wanted to raise the point to see if you considered this and made a decision or if the current name is for legacy reasons.

Reply:
Personally I prefer lower-case file names. And at least [KDE](https://github.com/KDE/kwin/blob/196423d26761c252c3ed14180b2b884e8a5546ef/client.h#L61) seems to follow this convention. I would continue keeping the file names in lower case.

I found this question on Stack Overflow [C++ why people don't use uppercase in name of header files?](https://stackoverflow.com/questions/30503532/c-why-people-dont-use-uppercase-in-name-of-header-files) but the answer does not refer anything that would validate the position.

Reply:
Works for me!